### PR TITLE
Fix: misplaced border ncselect

### LINKED ISSUE #9445: Partial border for recipient select
### Steps to reproduce

1. Set up an account
2. Click *New message*
3. Focus the *To* field

### Expected behavior

Good design

### Actual behavior

image

### Mail app version

3.6.0 beta1

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

@GretaD @hamza221 how can we fix this?

### PR DESCRIPTION
**Pull request by hamza221**
Fix #9445 

| b | a |
|--------|--------|
| image |image |

### CODE REVIEW

**Comment by GretaD** [comment]
i thought the issue was that the dropdown not having a top border.

**Comment by hamza221** [comment]
> i thought the issue was that the dropdown not having a top border.

Yes, I realised that later after checking pre-vue8 , that's why I turned it into draft

Pre vue8 there was no border and no slot for no results

**Comment by hamza221** [comment]
image

**Comment by hamza221** [comment]
I left the No-options slot in 
image

**Comment by hamza221** [comment]
PHP-CS unrelate to this PR

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/mail/pull/9552

File: src/components/Composer.vue [hunk 1]
```diff
@@ -96,7 +97,7 @@
 					:filter-by="(option, label, search)=>filterOption(option, label, search,'cc')"
 					:taggable="true"
 					:close-on-select="true"
-
+					::append-to-body="false"
```
Comment (GVodyanov) [inline src/components/Composer.vue:100]: Typo

**Review by GVodyanov — APPROVED** [review]
Works!

**Review by GretaD — COMMENTED** [review]
when i setup an account for the first time and i have no search results, it is shown like this. 
If i have at least one search result, it is shown correctly.

Screenshot from 2024-04-19 11-21-10

**Comment by ChristophWurst** [comment]
The borderless design does not look very appealing. Could we fully switch to the default ncselect styling with borders? We have bordered and stacked inputs in other places too: 
image

So I guess it would be fine to also have the borders in the Mail composer. @jancborchardt objections? if so, please provide mockups for a borderless select component

**Comment by hamza221** [comment]
@nextcloud/designers any suggestions?

**Review by marcoambrosini — COMMENTED** [review]
I think it makes sense to use our components as they natively look as @ChristophWurst suggests. Border-with will decrease to 1 pixel with the next release so they won't look as prominent. 

If with time we realise that it's still too much, I would prefer finding a solution upstream, since these dense forms are ubiquitous in Nextcloud.

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**